packages/core/src/ceramic.ts

```diff
@@ -121,8 +122,15 @@ export interface CeramicParameters {
   gateway: boolean,
   networkOptions: CeramicNetworkOptions,
   validateStreams: boolean,
+  syncOverride: SyncConfig
 }
 
+/**
+ * Global Sync Config Options
+ */
+export interface SyncConfig {
```

Comment:
why do we need this extra interface?  Couldn't `syncOverride` just be a `SyncOptions`?

Reply:
If you're trying to future-proof to a world where we may want to add other options to the override, such as `syncTimeoutSeconds` for example, you could have this be a `LoadOpts` and call the field `loadOptsOverride`

Reply:
Yeah was leaning towards that, just change/updated. Basically imagine options as something like this after
```
opts = { ...DEFAULT_CREATE_FROM_GENESIS_OPTS, this._loadOptsDefault, ...opts, ...this._loadOptsOverride}
```